### PR TITLE
feat: add hover transition states

### PR DESCRIPTION
**Pull request by simboonlong**
feat: extend color to themes
feat: add footer
refactor: colors to use from extended theme
chore: upgrade tailwind version

### CODE REVIEW

File: src/components/displayDocuments/documentCard.tsx [hunk 1]
```diff
@@ -17,10 +17,11 @@ const BackButton = styled.div`
 const CardContainer = styled.div`
   padding: 24px;
   box-shadow: 0px 0px 2px rgba(0, 0, 0, 0.04);
-  transition: box-shadow 0.4s ease-out;
+  transition: box-shadow 0.4s 0.1s ease-out, transform 0.3s ease-in-out;
 
   &:hover {
     box-shadow: 0px 0px 20px rgba(0, 0, 0, 0.3);
+    transform: translateY(-2px);
```
Comment (Nebulis) [inline src/components/displayDocuments/documentCard.tsx]: feels a bit too much to me 😅 but just my opinion, should ask feedback from other

File: tailwind.config.js [hunk 2]
```diff
@@ -1,6 +1,26 @@
 module.exports = {
   theme: {
-    extend: {}
+    textColor: theme => ({
```
Comment (Nebulis) [inline tailwind.config.js]: that's cool =), did you go through thiswith ben ? might be interesting for him

File: src/components/footer.tsx [hunk 3]
```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import { containerMain } from "./../constants";
```
Comment (Nebulis) [inline src/components/footer.tsx]: huh ? this sounds weird ?

File: src/components/displayDocuments/documentCard.tsx (hunk 1 above)
Comment (simboonlong) [reply src/components/displayDocuments/documentCard.tsx]: am ok with or without. ok i drop it for now.

File: tailwind.config.js (hunk 2 above)
Comment (simboonlong) [reply tailwind.config.js]: yup he's aware

File: src/components/footer.tsx (hunk 3 above)
Comment (simboonlong) [reply src/components/footer.tsx]: not sure what to name, changed it to cssContainerWrapper.

File: tailwind.config.js (hunk 2 above)
Comment (benjaminlim00) [reply tailwind.config.js]: yes i saw this

**Pull request merged**